client/asset/btc/spv_wrapper.go

```diff
@@ -158,10 +163,14 @@ func (s *btcChainService) Peers() []SPVPeer {
 	return peers
 }
 
+func (s *btcChainService) AddPeer(addr string) error {
+	return s.ChainService.ConnectNode(addr, true)
+}
+
 var _ SPVService = (*btcChainService)(nil)
 
 // BTCWalletConstructor is a function to construct a BTCWallet.
-type BTCWalletConstructor func(dir string, cfg *WalletConfig, chainParams *chaincfg.Params, log dex.Logger) BTCWallet
+type BTCWalletConstructor func(dir string, cfg *WalletConfig, chainParams *chaincfg.Params, log dex.Logger) (BTCWallet, error)
```

Comment:
I think we decided we could keep this signature error-free, right?

Reply:
Yeah I forgot to update this part.